package.json

```diff
@@ -123,9 +123,9 @@
     "not dead"
   ],
   "gitHooks": {
-    "post-checkout": "git submodule update --init --remote --recursive && yarn install",
-    "post-merge": "git submodule update --init --remote --recursive && yarn install",
-    "pre-commit": "git submodule update --init --remote --recursive && git add src/quacs-rs src/store/data && yarn install && lint-staged"
+    "post-checkout": "yarn install",
```

Comment:
We weren't doing this before, but this should be `yarn install --frozen-lockfile` I think.

Reply:
Updated with latest push